Python/Product/PythonTools/PythonTools/Debugger/DebugAdapter/DebugAdapterRemoteProcess.cs

```diff
@@ -59,7 +59,6 @@ internal sealed class DebugAdapterRemoteProcess : ITargetHostProcess, IDisposabl
                     _stream = new DebugRemoteAdapterProcessStream(new NetworkStream(socket, ownsSocket: true));
                     _stream.Disconnected += OnDisconnected;
                     _stream.Initialized += OnInitialized;
-                    _stream.LegacyDebugger += OnLegacyDebugger;
```

Comment:
This event comes from `DebugAdapterRemoteProcess.ConnectSocket` to inform user if they are connecting to a remote machine where the legacy ptvsd is running in the remote process, ie. they did an `import ptvsd; ...; ptvsd.wait_for_attach()` instead of `import debugpy; ...; debugpy.wait_for_client();`

It does that by checking the name of the messages it is receiving, it's easy because the legacy messages start with 'legacy'.

If you feel we don't need to worry about that anymore, then we should delete the code in DebugAdapterRemoteProcess that looks for the legacy messages and stop firing that LegacyDebugger event.